cmd/controller/main.go

```diff
@@ -447,10 +453,12 @@ type httpServer struct {
 
 func whenLeader(ctx context.Context, cancel context.CancelFunc, logger *logrus.Entry, doLeaderElection bool, kubeClient *kubernetes.Clientset, start func(_ context.Context)) {
 	if !doLeaderElection {
+		logger.Info("Leader election disabled!")
```

Comment:
I don't think we need these logs, we dump the `ctlConf` early in logs, e.g.:

```
{"ctlConf":{"MinPort":7000,"MaxPort":8000,"SidecarImage":"gcr.io/zml-gke-dev/agones-sdk:1.31.0-d98cd1b","SidecarCPURequest":"30m","SidecarCPULimit":"0","SidecarMemoryRequest":"0","SidecarMemoryLimit":"0","SdkServiceAccount":"agones-sdk","AlwaysPullSidecar":true,"PrometheusMetrics":true,"Stackdriver":false,"StackdriverLabels":"","KeyFile":"/home/agones/certs/server.key","CertFile":"/home/agones/certs/server.crt","KubeConfig":"","GCPProjectID":"","NumWorkers":100,"APIServerSustainedQPS":400,"APIServerBurstQPS":500,"LogDir":"/home/agones/logs","LogLevel":"debug","LogSizeLimitMB":10000,"AllocationBatchWaitTime":500000000,"LeaderElection":false},"featureGates":"CountsAndLists=false\u0026CustomFasSyncInterval=true\u0026Example=false\u0026FleetAllocationOverflow=false\u0026PlayerAllocationFilter=false\u0026PlayerTracking=false\u0026PodHostname=false\u0026ResetMetricsOnDelete=false\u0026SDKGracefulTermination=true\u0026SafeToEvict=true\u0026SplitControllerAndExtensions=true\u0026StateAllocationFilter=true","message":"starting gameServer operator...","severity":"info","source":"main","time":"2023-03-31T16:39:34.126611076Z","version":"1.31.0-d98cd1b"}
```

Reply:
done!